### PR TITLE
Fixed button group spacing of dropdown button 

### PR DESCRIPTION
**Pull request by prakhar1535**
#7386

### CODE REVIEW

**Comment by ehconitin** [comment]
@prakhar1535 that was fast! haha ;)

**Comment by ehconitin** [comment]
@prakhar1535 ,
Thanks for the PR! However, please avoid making changes to the core UI components, as this could impact other buttons across the project. Instead, you can style the specific button mentioned in the issue using `@emotion/styled` and adjust its padding accordingly! :)

**Comment by prakhar1535** [comment]
@ehconitin fixed the issue it does not interfare the core ui component.

File: packages/twenty-front/src/modules/views/components/UpdateViewButtonGroup.tsx [hunk 1]
```diff
@@ -27,7 +27,9 @@ const StyledContainer = styled.div`
   margin-right: ${({ theme }) => theme.spacing(2)};
   position: relative;
 `;
-
+const DropdownButton = styled(Button)`
```
[the remark this answers is not in the document]
Comment (prakhar1535) [reply packages/twenty-front/src/modules/views/components/UpdateViewButtonGroup.tsx:30]: oh it wasnt in mine but let me fix this super fast

[the remark this answers is not in the document]
Comment (prakhar1535) [reply packages/twenty-front/src/modules/views/components/UpdateViewButtonGroup.tsx]: @ehconitin can you verify it now ?

**Review by ehconitin — APPROVED** [review]
LGTM! Thanks for contributing!

**Comment by prakhar1535** [comment]
> LGTM! Thanks for contributing!

Thanks for corrections and merging :)

**Pull request merged**